python/dask_cudf/dask_cudf/io/tests/test_json.py

```diff
@@ -4,11 +4,12 @@
 import pytest
 
 import dask
-import dask.dataframe as dd
 from dask.utils import tmpfile
 
 import dask_cudf
 
+import dask.dataframe as dd  # isort:skip
```

Comment:
Why the override here?

Reply:
Now that gpuCI's `isort==5.0.7` check is targetting this file, it want to change this line to

```python
from dask import dataframe as dd
```

I couldn't find an option for `isort` to allow this type of import, but can confirm that `isort==5.6.4` (which we are planning to bump gpuCI to after this PR is merged) allows either style of import, so this override can be removed once rapidsai/integration#286 is merged.